robot/EDAP/tests/api/reciprocal_relationship_delete.robot

```diff
@@ -1,6 +1,6 @@
 *** Settings ***
 
-Resource  robot/HEDAP/resources/HEDA.robot
+Resource  robot/EDAP/resources/EDA.robot
```

Comment:
@dashattuck Same question applies here and the files below re: renaming to "EDA" instead of "EDAP"

Reply:
I'd rather it be EDA, so that we drop the P 

I'm not sure how that folder gets created (HEDAP).  Is it based on a setting in cumulusci.yml?

Reply:
I'm not sure, either. Is it not just a matter of updating the folder name and then all references in the robot files to point to the new folder name?

Reply:
Correct. The directory name and any references to it must be renamed.

Reply:
ok - I'll modify my 'resource' paths to be:

Resource        robot/EDA/resources/EDA.robot

Reply:
@robertbwatson, @jstvz - have we notified SAL of these changes?  Do they have any dependencies on the naming convention HEDA?

Reply:
Does SAL reference our robot tests? I would have expected that the robot tests are project-specific, but it'd be great to verify. Everyone should be aware that the EDA changes are coming as it's been announced internally. I'll ping the EDU room tomorrow as a reminder.

Reply:
No. While cross-product robot references is on the roadmap I haven't written the necessary `cci` task to import libraries from another project yet. As of today, no projects should be directly relying on the robot directory structure.

Reply:
Doug made a good point that there might be other robot tests that look for the "HEDA Settings" or "HEDA" text, so it will be good to watch.